draft-ietf-teep-architecture.md

```diff
@@ -1347,6 +1347,23 @@ still a threat present in allowing a potentially compromised TAM to install
 any TA it wants; for example, a known-compromised TA or an explicitly
 malicious TA.
 
+## Hostile TAM
+
+There are also threats of hostile or abusive TAMs where a TAM is compromised and thus turns to act out of 
+expectation of Device Administrators, for example, pushing out TAs that contain 
+some data collection or use users' device resources for distributed jobs for a TAM. 
+To mitigate this threat, TEEs and Device Administrators have several options for 
+detecting and mitigating a hostile TAM:
+
+1. Trust Anchor update: when a Trust Anchor entry in the TEE Trust Anchor list is specific to 
+the hostile TAM, a Trust Anchor update process may be used to remove the Trust Anchor as a way 
+to stop the TAM from installing TAs to the TEE.
+2. TAM certificate revocation: the certificate issuing authority of the TAM certificate is 
+requested to revoke the TAM's certificate. TEEP Agents that support a certificate revocation 
+status check will be able to stop the TAM, and further decide to remove those hostile TAs from that TAM.
+3. TA removal by the Device Administrator upon detection of a hostile TAM and any abusive TA.
+The detection can be the result of 1) above, and some other out-of-band detection technique.
```

Comment:
My opinion: lines 1339-1343 and lines 1358-1365 require confirming consensus if left in, and have some risk of objections since they are not simply editorial "things an admin should think about"

Lines 1336-1337 can even be (mis?)read as implying the list is a complete list and there cannot be other options.

Reply:
I see your point ""things an admin should think about"" - Trust Anchor update may not be familiar to an ordinary Device Administrator which can be a device owner for the knowledge gap.

Device Administrators may contact "some" organization to stop or remove a TAM that turns to be bad, or contact the TAM to just stop it which can work in some cases, as to how we handle real world scenarios with social networking companies.

Reply:
@dthaler please review the updated paragraphs.